pkg/dao/domains.go

```diff
@@ -32,8 +41,13 @@ func (dDao domainDaoImpl) FetchOrCreateDomain(ctx context.Context, orgId string)
 }
 
 func (dDao domainDaoImpl) Create(ctx context.Context, orgId string) (string, error) {
+	name := fmt.Sprintf("cs-%v", random.New().String(10, random.Hex))
```

Comment:
didn't notice in the previous PR but old custom domains (prefixed with cs-) have a length of 11 and new ones have a length of 13. my guess is that they were originally set to the same length as an org id? just wanted to point this out, do you think it would cause any issues?

Reply:
i'm assuming it shouldn't matter since the new red hat domain is also a different length 😄

Reply:
yeah, i think it was just chosen to be small, but not too small.  The actual size doesn't matter too much